### PR TITLE
Typos desaign

### PR DESCRIPTION
**Pull request by GoralTomaszGorski**
Hallo my Mentor!
I have improved some templates,
CSS, design. I
 would appreciate for your feedback on this.

### CODE REVIEW

File: src/main/java/goral/psychotherapistoffice/web/admin/PatientManagementController.java [hunk 1]
```diff
@@ -28,12 +27,12 @@ public String addPatientFrom(Model model){
         PatientDto patientDto = new PatientDto();
         model.addAttribute("patientDto", patientDto);
         model.addAttribute("heading",
-                "Podaj swoje dane, aby umówić wizytę");
+                "Podaj dane pacjęta, aby umówić wizytę");
         model.addAttribute("description",
-                "Twoje dane będzie widział jedynie terapeuta. Ogólnodostępne będzie jednynie NICK. ");
+                "Dane będzie widział jedynie terapeuta. Ogólnodostępne będzie jednynie NICK. ");
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/web/admin/PatientManagementController.java:32]: Ogólnodostępne -> ogólnodostępny

File: src/main/java/goral/psychotherapistoffice/web/AboutMeController.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ public class AboutMeController {
 
     @GetMapping("/omnie")
     public String therms(Model model){
-        model.addAttribute("heading", "Ewa Górska psychoterapełta, psycholog, mediator");
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/web/AboutMeController.java:13]: i like it

File: src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java [hunk 3]
```diff
@@ -27,7 +27,7 @@ public String addTherapyForm(Model model) {
         TherapyDto therapyDto = new TherapyDto();
         model.addAttribute("therapyDto", therapyDto);
         model.addAttribute("heading",
-                "Dodaj rodzaj terapii, opis, cenę. Nie wpisuj Id, zostanie wygenerowane autoamtycznie");
+                "Dodaj rodzaj terapii, opis, cenę. Nie wpisuj Id, zostanie wygenerowane automatycznie.");
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java]: "Nie wpisuj Id" maybe should we hide id input?

File: src/main/resources/templates/user/mail-form.html [hunk 4]
```diff
@@ -8,22 +8,49 @@
...
+
+                <div class="mar">
+                    <h4>Ewa Górska</h4>
+                    <p>ul. Fabryczna 6, 23-200 Kraśnik</p>
+                    <p>tel (+48) 781 106 026</p>
+                    <p>nr rachunku bankowego</p>
+                    <p>BKO BP: rachunek do sprawdzenia</p>
+                    <p>57 1020 1185 0000 4002 0117 8185</p>
+                    <div class="pic flip-box2">
+                        <img src="/images/mapka.jpg"
```
Comment (gorskimarcin96) [inline src/main/resources/templates/user/mail-form.html]: you can replace image to google map: 

```html
<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1135.9092467502423!2d22.175846920938483!3d50.96466213740499!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4722e965253a8e77%3A0x818431a83eb44f8d!2sPrzychodnia%20Fabryki%20%C5%81o%C5%BCysk%20Tocznych%20Kra%C5%9Bnik%20S.A.!5e0!3m2!1sen!2spl!4v1713379199803!5m2!1sen!2spl" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```

image

image

**Review by gorskimarcin96 — APPROVED** [review]

File: src/main/resources/templates/user/mail-form.html (hunk 4 above)
Comment (gorskimarcin96) [reply src/main/resources/templates/user/mail-form.html]: you can check how working my map: https://mgorski.dev/pl/kontakt

File: src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java (hunk 3 above)
Comment (GoralTomaszGorski) [reply src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java]: I don't wont create new template for edit add ad therapy so I add only this text in controller and use the same templete;) 
I will fix it:)

File: src/main/resources/templates/user/mail-form.html (hunk 4 above)
Comment (GoralTomaszGorski) [reply src/main/resources/templates/user/mail-form.html]: łooo a to dobre
It is beautiful szwagier:)
I don't know the function

File: src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java (hunk 3 above)
Comment (gorskimarcin96) [reply src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java]: we can use same template and add if else condition

**Comment by GoralTomaszGorski** [comment]
I only saw today that I can correct the code from  InteliJ level and your comment is visible.
Better late than never.
:)

**Pull request merged**